internal/infrastructure/datastore/db.go

```diff
@@ -58,6 +64,7 @@ func NewDbMigration(config *config.Config) (db *gorm.DB, m *migrate.Migrate, err
 	if err != nil {
 		return nil, nil, fmt.Errorf("could not connect to database: %w", err)
 	}
+	sqlDb.SetMaxOpenConns(config.Database.MaxOpenConns)
```

Comment:
Same as the above suggestion.

```suggestion
	slog.Info("Setting database max connections",
		slog.Int("Database.MaxOpenConns", config.Database.MaxOpenConns),
	)
	sqlDb.SetMaxOpenConns(config.Database.MaxOpenConns)
```